Packs/Gatewatcher-AionIQ/Integrations/GCenter/README.md

```diff
@@ -1,284 +1,26 @@
 This integration allows, via about twenty commands, to interact with the GCenter appliance via its API.
 This integration was integrated and tested with version v2.5.3.102 of GCenter.
-To simplify GCenter v2.5.3.102 is called GCenter in the Pack.
+To simplify GCenter v2.5.3.102 is called GCenterv102 in the Pack.
```

Comment:
Was the name changes anywhere else? if there is a requirement to create a v2 integration? I don't think it's needed by the code changes.

Reply:
its simply a mistake because the integration was previously called GCenterv102